src/styles/blocks/card.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    height: 134px;
+    width: 160px;
+    margin: 0 auto 40px;
+  }
+
+  &__name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-text-color;
+    padding-bottom: 4px;
+  }
+
+  &__code {
+    font-weight: 400;
```

Comment:
It's default value in this property. Remove redundant code

Reply:
fix it everywhere
https://github.com/mate-academy/layout_catalog/pull/2555/files#r1066236329